common/src/api/external/error.rs

```diff
@@ -292,28 +411,42 @@ impl From<Error> for HttpError {
                 internal_message,
             },
 
-            Error::InvalidRequest { message } => HttpError::for_bad_request(
-                Some(String::from("InvalidRequest")),
-                message,
-            ),
+            Error::InvalidRequest { message } => {
```

Comment:
(applies to this whole function)

aside from the internal message, are any of these changing at all?

Reply:
No, no changes.